src/Button/Button.js

```diff
@@ -1,56 +1,55 @@
+import { buttonVariants, buttonSizes, buttonTypes } from './Button.levers';
 
 /**
  * Button Component
  *
  * HTML button element
  */
-export const Button = ({ className, children, onClick, ref, ...props }) => {
+export const Button = ({ className, children, onClick, ref, variant, size, type, isDisabled, ...props }) => {
   // Defaults & Variables.
   // ---------------------------------------------------------------------------
-  const classnames = require('classnames')
-  const levers = {}
+  const classnames = require('classnames');
+  const levers = {};
 
   // Levers
   // ---------------------------------------------------------------------------
 
   // Props.variant
-  if (props.variant && buttonVariants.includes(props.variant)) {
-    switch (props.variant) {
+  if (variant && buttonVariants.includes(variant)) {
+    switch (variant) {
       case 'primary':
-        levers.variant = classnames('su-bg-digital-red su-text-white su-border-2 su-border-digital-red su-border-solid hover:su-border-black focus:su-border-black')
-        break
+        levers.variant = classnames('su-bg-digital-red su-text-white su-border-2 su-border-digital-red su-border-solid hover:su-border-black focus:su-border-black');
+        break;
 
       case 'secondary':
-        levers.variant = classnames('su-bg-transparent hover:su-bg-transparent focus:su-bg-transparent su-text-digital-red hover:su-text-black focus:su-text-black su-border-2 su-border-digital-red su-border-solid hover:su-border-black focus:su-border-black')
-        break
+        levers.variant = classnames('su-bg-transparent hover:su-bg-transparent focus:su-bg-transparent su-text-digital-red hover:su-text-black focus:su-text-black su-border-2 su-border-digital-red su-border-solid hover:su-border-black focus:su-border-black');
+        break;
 
       case 'none':
-        levers.variant = classnames('su-bg-transparent hover:su-bg-transparent focus:su-bg-transparent')
-        break
+        levers.variant = classnames('su-bg-transparent hover:su-bg-transparent focus:su-bg-transparent');
+        break;
     }
   }
 
   // Props.size
-  if (props.size && buttonSizes.includes(props.size)) {
-    switch (props.size) {
-      case 'default':
-        levers.size = classnames('su-px-26 su-py-10 su-text-20')
-        break
-
+  if (size && buttonSizes.includes(size)) {
+    switch (size) {
       case 'big':
-        levers.size = classnames('su-px-34 su-py-15 su-text-22')
-        break
+        levers.size = classnames('su-px-34 su-py-15 su-text-22');
+        break;
 
       case 'small':
-        levers.size = classnames('su-px-19 su-py-9 su-text-18')
-        break
+        levers.size = classnames('su-px-19 su-py-9 su-text-18');
+        break;
 
       case 'minimal':
-        levers.size = classnames('su-p-0')
-        break
+        levers.size = classnames('su-p-0');
+        break;
+
+      default:
+        levers.size = classnames('su-px-26 su-py-10 su-text-20');
```

Comment:
Moved default to the default so you don't have to pass in the keyword `default`.